### ISSUE TITLE
<meta name="twitter:card"...> (and similar) entities trip up the parser [rt.cpan.org #132922]

### ISSUE THREAD

**Issue by oalders** [libwww-perl/HTML-Parser#8]
Migrated from [rt.cpan.org#132922](https://rt.cpan.org/Ticket/Display.html?id=132922) (status was 'new')

Requestors:
* fleetfootmike@gmail.com

From fleetfootmike@gmail.com on 2020-07-01 15:53:28
:
```
Look alike it calls HTTP::Headers->push_header() with a constructed header name of X-Meta-Twitter:card, which obviously fails. 
```

From fleetfootmike@gmail.com on 2020-07-01 15:55:58
:
```

> Looks like it calls HTTP::Headers->push_header() with a constructed header name of X-Meta-Twitter:card, which obviously fails. 

Where 'it', obviously, its HTML::HeadParser - sorry for being unclear.

```

**Comment by oalders** [comment]
@haarg so is the assumption that the person who opened this ticket was on a very old version `HTML::Parser`?

**Comment by haarg** [comment]
Actually I was wrong about the version this was fixed in. It was fixed in v3.71.

It does seem likely that the reporter of this was using an old version.

The original ticket for this issue: https://rt.cpan.org/Ticket/Display.html?id=80524

**Comment by oalders** [comment]
Thanks for the research!
